.github/workflows/pipeline-deploy-app-preprod-prod.yaml

```diff
@@ -96,15 +97,15 @@ jobs:
     secrets: inherit
 
   slack-notifications:
-    name: Send notification to slack
+    name: Send Notification to Slack
```

Comment:
Interesting .. we are inconsistent here some are in quotes some are not. To avoid a change and then asking you to review again I am going to leave this but note that we should maybe be consistent - thanks @AITR1